apps/gallery/test/marionette/apps/fakeactivitycaller/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Fake Gallery Activity Caller</title>
```

Comment:
This app is both a caller and a handler now, and it isn't actually faking anything: it is initiating and handling real activities. So this app name isn't quite right. It may not be worth the trouble to change, but you could call it "Gallery Activity Tester" or something like that.